test/e2e/flask/multichain-api/create-session.spec.ts

```diff
@@ -261,4 +261,144 @@ describe('Multichain API', function () {
       });
     });
   });
+
+  describe('Connect wallet to the multichain dapp via `externally_connectable`, call `wallet_createSession` without any accounts requested', function () {
+    it('should automatically select first account', async function () {
```

Comment:
```suggestion
    it('should automatically select the first account', async function () {
```

Reply:
resolved in [6e0626b](https://github.com/MetaMask/metamask-extension/pull/29400/commits/6e0626b85a265db8e150209a2ae91b1a1f18ee04)